src/Network/Ethereum/Web3/Transaction.hs

```diff
@@ -0,0 +1,62 @@
+module Network.Ethereum.Web3.Transaction where
+
+import           Crypto.Hash                       (Digest, Keccak_256, hash)
+import           Crypto.Secp256k1
+import           Data.ByteArray                    (Bytes, convert)
+import           Data.ByteString
+import qualified Data.ByteString                   as BS
+import           Data.ByteString.Short             (fromShort)
+import qualified Data.ByteString.Base16            as BS16
```

Comment:
`memory` library also exports `convertFromBase`: https://hackage.haskell.org/package/memory-0.14.16/docs/Data-ByteArray-Encoding.html and additional library isn't needed.